packages/storybook/src/decorators/withStrictMode.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { useStrictMode } from '../knobs/useStrictMode';
+
+const StrictModeWrapper: React.FunctionComponent<{}> = props => {
+  const strictMode = useStrictMode();
+  return strictMode ? <React.StrictMode>{props.children}</React.StrictMode> : <>{props.children}</>;
+};
+
+export const withStrictMode = (storyFn: () => React.ReactNode) => {
```

Comment:
I'm not entirely sure about duplicating this, since v9 vr tests still run on converged, so duplicating (and potentially diverging) SB infra could be a risk. I would prefer if we only duplicated once we break v9 vr-tests completely. I won't block your PR on this though

Reply:
The withStrictMode decorator may be present in vr-tests due to config inheritance, but it's never enabled. It wouldn't be useful in vr-tests anyway since it only logs console warnings.